hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/ReduceTask.java

```diff
@@ -62,7 +62,7 @@
 @InterfaceStability.Unstable
 public class ReduceTask extends Task {
 
-  static {                                        // register a ctor
+  static {                                        // register actor
```

Comment:
Not a typo I believe: "register a ctor" is short for "register a constructor"

Reply:
Thx for review. That makes a lot of sense.